src/ontology/cl-edit.owl

```diff
@@ -29018,6 +29020,17 @@ SubClassOf(obo:CL_4023025 ObjectSomeValuesFrom(obo:RO_0002162 obo:NCBITaxon_1009
 SubClassOf(obo:CL_4023025 ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0061534))
 SubClassOf(obo:CL_4023025 ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_P60041))
 
+# Class: obo:CL_4023026 (direct pathway medium spiny neuron)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:34727523"^^xsd:string) Annotation(oboInOwl:hasDbXref "WikipediaVersioned:Medium_spiny_neuron&oldid=1109726850"^^xsd:string) obo:IAO_0000115 obo:CL_4023026 "A medium spiny neuron that expresses dopamine type 1 receptors and projects to the globus pallidus internus or the substantia nigra pars reticulata."^^xsd:string)
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4023026 <https://orcid.org/0000-0001-7258-9596>)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:34727523"^^xsd:string) Annotation(oboInOwl:hasSynonymType <http://purl.obolibrary.org/obo/cl#abbreviation>) oboInOwl:hasExactSynonym obo:CL_4023026 "D1-MSN")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:30096299"^^xsd:string) Annotation(oboInOwl:hasSynonymType <http://purl.obolibrary.org/obo/cl#abbreviation>) oboInOwl:hasExactSynonym obo:CL_4023026 "dSPN"^^xsd:string)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:34727523"^^xsd:string) oboInOwl:hasExactSynonym obo:CL_4023026 "dopamine 1 medium spiny neuron")
+AnnotationAssertion(rdfs:label obo:CL_4023026 "direct pathway medium spiny neuron")
+SubClassOf(obo:CL_4023026 obo:CL_1001474)
+SubClassOf(obo:CL_4023026 ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_000001175))
```

Comment:
May be an upstream modelling issue, but is there evidence D1A receptors are expressed (versus any D1 receptor)? Uberon shows D1 receptor and D1A receptors as exact synonyms... unsure if that is accurate.

Reply:
It is accurate according to PR ontology definition (translation of DRD1) where it is imported from - DRD1 encodes for dopamine 1 receptors. I think the modelling is right as it groups D1-like together (which includes DRD5). Took me a second look when I was adding that too and I studied dopamine receptors for my PhD >.< 2 years and everything is forgotten lol